src/vmm/src/resources.rs

```diff
@@ -138,7 +138,7 @@ impl VmResources {
         let vmm_config = serde_json::from_str::<VmmConfig>(config_json)?;
 
         if let Some(logger) = vmm_config.logger {
-            init_logger(logger, instance_info)?;
+            logger.apply()?;
         }
```

Comment:
I think directly using `LOGGER` to apply config is better:
```rust
if let Some(logger_config) = vmm_config.logger_config {
    LOGGER.apply(logger_config)?;
}
```

Reply:
Updated to include this.